app/views/subjects/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+= form_for @subject do |f|
+  .field
+    = f.label :name
+    = f.text_field :name
+  .field
+    = f.label :desc
+    = f.text_area :desc
+  .actions
+    = f.submit
+  - if @subject.errors.any?
+    #error_explaination
+    %h2= pluralize(@subject.errors.count, 'error')
```

Comment:
Yeah, it's inside in the ERB so I would indent it to put it inside in the new version.